### DISCUSSION TITLE
Visualizing different labels within the same label tree in object detection workflow

### DISCUSSION THREAD

**Discussion by emmeb2021** [biigle/core#1003] [Q&A] [answered]
Hi Guys
I am testing different object detection models on imagery and feeding the output here :projects/3315
I am only so far running this in volumes/20768, the annotations are made using models with slightly adjusted parameters. I want to be able to quickly check the output visually, but of course there is overlap between the model box coordinates. 
Is there any way to visualize one label at a time?
Cheers
Emme

**Comment by mzur** [comment, marked as answer]
Hi Emme, so if I understand correctly, you would like to switch between annotations of different labels? Currently, you can use the filter in the annotations tab to show only annotations with a specific label. This won't be very convenient to quickly switch between labels to compare the detections, though. Maybe you could create a screenshot of the annotations of each label (with the button in the settings tab) and then compare these?

**Reply by emmeb2021** [reply]
Sorry im blind! Yep the filter label tool is exactly what I needed! cheers, Em
